submissions/tests/test_api.py

```diff
@@ -183,12 +185,14 @@ def test_create_non_json_answer(self):
             api.create_submission(STUDENT_ITEM, datetime.datetime.now())
 
     def test_load_non_json_answer(self):
-        # This should never happen, if folks are using the public API.
-        # Create a submission with a raw answer that is NOT valid JSON
         submission = api.create_submission(STUDENT_ITEM, ANSWER_ONE)
         sub_model = Submission.objects.get(uuid=submission['uuid'])
-        sub_model.raw_answer = ''
-        sub_model.save()
+
+        # This should never happen, if folks are using the public API.
+        # Create a submission with a raw answer that is NOT valid JSON
+        query = "UPDATE submissions_submission SET raw_answer = '}' WHERE id = %s"
+        connection.cursor().execute(query, [str(sub_model.id)])
+        transaction.commit_unless_managed()
```

Comment:
Actually this is removed in django 1.8. Can we use something else but i did't get why we are using this at all.